### PR TITLE
Language Detector task

### PR DESCRIPTION
**Pull request by craiglabenz**
## Description

Adds the `LanguageDetector.detect(...)` task.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I read and followed the [Flutter Style Guide].
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

[issue database]: https://github.com/google/flutter-mediapipe/issues
[Contributor Guide]: https://github.com/google/flutter-mediapipe/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by craiglabenz** [comment]
Apologies for taking a minute to fix that merge conflict, @Piinks, but this should now be ready.

File: packages/mediapipe-task-text/lib/src/interface/tasks/language_detection/language_detection.dart [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```
Comment (Piinks) [inline packages/mediapipe-task-text/lib/src/interface/tasks/language_detection/language_detection.dart:4]: Should this be a `library`?

File: packages/mediapipe-task-text/lib/src/interface/tasks/language_detection/language_detector.dart [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'language_detection.dart';
+
+/// {@template LanguageDetector}
+/// {@endtemplate}
```
Comment (Piinks) [inline packages/mediapipe-task-text/lib/src/interface/tasks/language_detection/language_detector.dart:8]: Can this empty template be removed? Or filled with intended doc? :)

File: packages/mediapipe-task-text/lib/src/io/tasks/language_detection/language_detection.dart [hunk 3]
```diff
@@ -0,0 +1,8 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```
Comment (Piinks) [inline packages/mediapipe-task-text/lib/src/io/tasks/language_detection/language_detection.dart:4]: Same here re: library

File: packages/mediapipe-task-text/lib/src/io/tasks/language_detection/language_detector_executor.dart [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+/// Executes MediaPipe's "detect language" task.
+///
+/// {@macro TaskExecutor}
+class LanguageDetectorExecutor extends TaskExecutor<
+    bindings.LanguageDetectorOptions,
+    LanguageDetectorOptions,
+    bindings.LanguageDetectorResult,
+    LanguageDetectorResult> {
+  /// {macro LanguageDetectorExecutor}
```
Comment (Piinks) [inline packages/mediapipe-task-text/lib/src/io/tasks/language_detection/language_detector_executor.dart:23]:
```suggestion
  /// {@macro LanguageDetectorExecutor}
```

File: packages/mediapipe-task-text/lib/src/interface/tasks/language_detection/language_detection.dart (hunk 1 above)
Comment (craiglabenz) [reply packages/mediapipe-task-text/lib/src/interface/tasks/language_detection/language_detection.dart:4]: I don't think so. (My understanding is that) that's just an anchor for comments, but we don't have any here.

File: packages/mediapipe-task-text/lib/src/interface/tasks/language_detection/language_detector.dart (hunk 2 above)
Comment (craiglabenz) [reply packages/mediapipe-task-text/lib/src/interface/tasks/language_detection/language_detector.dart:8]: ha, whoops. Left myself a placeholder - time to fill it in!

**Pull request merged**